packages/convertkit/src/forms/subscribe/subscribe-to-convertkit-form.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react'
+import * as Yup from 'yup'
+import axios from 'axios'
+import {useFormik} from 'formik'
+import {Input, Button} from '@skillrecordings/react/dist/components'
+
+const SUBSCRIBE_API_URL =
+  process.env.NEXT_PUBLIC_SUBSCRIBE_API_URL || '/api/convertkit/subscribe'
+
+type ConvertkitSubscriber = {
+  id: string
+  email: string
+  first_name: string
+}
+
+type SubscribeFormProps = {
+  actionLabel?: string
+  successMessage?: string | React.ReactElement
+  errorMessage?: string | React.ReactElement
+  submitButtonElem?: React.ReactElement
+  onError?: (error?: any) => void
+  onSuccess?: (subscriber?: ConvertkitSubscriber) => void
+  formId: number
```

Comment:
Can this be optional since we're defaulting to `CONVERTKIT_SIGNUP_FORM` env var?